src/Microsoft.Tye.Core/KubernetesManifestGenerator.cs

```diff
@@ -229,7 +229,7 @@ internal static class KubernetesManifestGenerator
                                             { "secretKeyRef", new YamlMappingNode()
                                                 {
                                                     { "name", new YamlScalarNode(binding.Name) { Style = ScalarStyle.SingleQuoted } },
-                                                    { "key", new YamlScalarNode("connectionString") { Style = ScalarStyle.SingleQuoted } },
+                                                    { "key", new YamlScalarNode("connectionstring") { Style = ScalarStyle.SingleQuoted } },
```

Comment:
cross-checked this with: https://github.com/dotnet/tye/blob/master/src/Microsoft.Tye.Core/ValidateSecretStep.cs#L113

Reply:
Looks good 👍